doc/Administration-guide/System-components-overview/index.md

```diff
@@ -5,63 +5,73 @@ permalink: /:path/
 
+## Kaa node
 
-The *Kaa IoT platform* consists of the *Kaa server* and *endpoint SDKs*. The Kaa server implements the back-end part of the platform, exposes integration interfaces, and offers administrative capabilities. An endpoint SDK is a library which provides communication, data marshalling, persistence, and other functions available in Kaa for specific type of an endpoint (e.g. Java-based, C++-based, C-based, Objective-C-based). This SDK can be used to create Kaa clients, which are any pieces of software that utilize Kaa functionality and are installed on some connected devices. It is the responsibility of the Kaa client to process structured data provided by the Kaa server (configuration, notifications, etc.) and to supply data to the return path interfaces (profiles, logs, etc.).
-A *Kaa instance* (interchangeable with *Kaa deployment*) is a particular implementation of the *Kaa platform* and it consists of a Kaa cluster and *endpoints*. A Kaa cluster represents a number of interconnected Kaa server *nodes*. An *endpoint* is an abstraction which represents a separate managed entity within a Kaa deployment. Practically speaking, an endpoint is a specific Kaa client registered (or waiting to be registered) within a Kaa deployment. For example, a news application installed on your mobile phone, the same news application installed on your tablet, and the same news application on your WiFi-enabled fridge would be considered three different endpoints in Kaa.
-An *application* in Kaa represents a family of available implementations of a specific software application used by endpoints. For example, two versions of a sound frequency measuring application which differ by their implementation for, respectively, Arduino and STM32 platforms would be considered the same application in Kaa.
-A tenant in Kaa is a separate business entity which contains its own endpoints, applications and users.
-A single Kaa deployment is able to support multiple tenants, with multiple applications per each tenant. As illustrated in the following diagram, applications belong to tenants, while endpoints register within applications. In addition to tenants, applications and endpoints, there are also users and endpoint groups, which will be described in detail further in this reference.
+Each Kaa node in cluster runs combination of Control, Operations, and Bootstrap services. 
+A system administrator can enable or disable any of the services on a particular node, thus flexibly configuring the cluster deployment (see [general configuration]({{root_url}}Administration-guide/System-Configuration/General-configuration/) for more details).
 
-![Kaa instance (cluster)](1-04.png "Kaa instance (cluster)")
+Kaa Control service is responsible for managing overall system data, processing API calls from the web UI and external integrated systems, and delivering corresponding notifications to Operations services.
```

Comment:
'processing API calls from the web UI and external integrated systems' - 'processing API calls from the Administration UI and external integrated systems communicating via REST API'